web3js-ext/src/web3/web3.ts

```diff
@@ -42,6 +52,29 @@ export class KlaytnWeb3 extends Web3 {
 
   // Below methods return a function bound to the context 'web3'.
 
+  accounts_create(context: Web3Context): typeof this.eth.accounts.create {
+    return (): Web3Account => {
+      return createWithContext(context);
+    }; 
+  }
+
+  accounts_privateKeyToAccount(context: Web3Context): typeof this.eth.accounts.privateKeyToAccount {
+    return (privateKey: Bytes, ignoreLength?: boolean): Web3Account => {
+      return privateKeyToAccountWithContext(context, privateKey, ignoreLength);
+    }; 
+  }
+
+  async accounts_decrypt(keystore: KeyStore | string, password: string, options?: Record<string, unknown>){
+		const account = await decrypt(keystore, password, (options?.nonStrict as boolean) ?? true);
+
+		return {
+			...account,
+      // // TO-DO : decrypt function will be implemented with KeyStore V4 later
```

Comment:
It's `TODO` to be precise.
Also, change the tabs to spaces please.

Reply:
Oops, thank you.
Updated in https://github.com/klaytn/web3klaytn/pull/522/commits/44bb9e29391b9e923ce719ae08e3ad7adb4c8fb0.